widgetssdk/src/main/java/com/glia/widgets/push/notifications/RequestPushNotificationDuringAuthenticationUseCase.kt

```diff
@@ -42,7 +46,16 @@ internal class RequestPushNotificationDuringAuthenticationUseCaseImpl(
             permissionManager.shouldShowPermissionRationale(Manifest.permission.POST_NOTIFICATIONS) -> {
                 // Show rationale dialog
                 uiComponentsDispatcher.showNotificationPermissionDialog(onAllow = {
+                    Logger.i(
+                        TAG,
+                        "Visitor's decision for intermediate push notification permission is 'allowed'"
```

Comment:
I also see the `Visitor's decision for push notification permission during auth. is 'allowed=$it'`. Does this mean that this event would be logged twice or these are two different places that will not happen at the same time?